core/src/main/java/org/elasticsearch/script/ScriptService.java

```diff
@@ -646,7 +645,17 @@ public ParseField getParseField() {
             return parseField;
         }
 
-        public ScriptMode getDefaultScriptMode() {
+        public ScriptMode getDefaultScriptMode(@Nullable String language) {
+            if (language != null) {
```

Comment:
That's basically what the sandbox option was used for previously. I don't expect us to change this at all, all scripting languages are disabled by default except for these three specifically.